src/mozillavpn.cpp

```diff
@@ -407,14 +416,14 @@ void MozillaVPN::maybeStateMain() {
         settingsHolder->setGleanEnabled(false);
```

Comment:
Is this bug being resolved in another ticket?

Reply:
[VPN-5520: All pre-onboarding glean data being deleted on first sign in](https://mozilla-hub.atlassian.net/browse/VPN-5520)

Reply:
We decided not to block on this, right?

Reply:
We are blocking turning the feature flag on, on that ticket